sqlobject/src/test/java/org/jdbi/v3/sqlobject/BindTime.java

```diff
@@ -34,18 +32,7 @@
     class Factory implements SqlStatementCustomizerFactory {
         @Override
         public SqlStatementCustomizer createForType(Annotation annotation, Class<?> sqlObjectType) {
-            return stmt -> stmt.bind("now", OffsetDateTime.now(stmt.getConfig(Config.class).clock));
-        }
-    }
-
-    class Config implements JdbiConfig<Config> {
```

Comment:
This is a breaking API change--inner classes of interfaces are implicitly public.

Reply:
This is in `test` of `sqlobject`

Reply:
whoops! carry on